docs/installation/mac.md

```diff
@@ -56,10 +57,37 @@ You can test if it's up and running by issuing the following command:
 redis-cli ping
 ```
 
+### Elasticsearch
+
+DEV requires Elasticsearch version 7 or higher.
+
+We recommend installing from archive, but if you prefer you can also use
+[Homebrew](https://brew.sh) to install Elasticsearch, directions at the bottom
+of this section. The following directions were
```

Comment:
maybe we can omit the `brew` installation as we know it's currently broken, I'm good either way